policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,85 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        year = period.start.year
+        p = parameters(period).gov.states.ar.tax.income.deductions.itemized
+        if year < 2018:
+            instant_str = f"2018-01-01"
+        else:
+            instant_str = f"{year}-01-01"
+        p_ded = parameters(instant_str).gov.irs.deductions
```

Comment:
why do we need this condition for the deductions.yaml file? I thought it only applies to the medical expenses rate

Reply:
Since for the itemized deductions list, it came into use only after 2018, I added this to cope with the problem of unit test for different periods for medical expenses rate. Did I understand what you mean incorrectly?